docker-compose.override.yml

```diff
@@ -21,46 +22,49 @@ services:
       - EMBED_API_TOKEN=${EMBED_API_TOKEN}
     depends_on:
       - api
+
   mongo:
     image: mongo
     networks:
       - hc-network
     tty: true
     command: "--smallfiles"
+
   api:
     image: humanconnection/api-feathers:edge
     depends_on:
       - maildev
       - thumbor
       - mongo
+    links:
+      - mongo:mongo.local
```

Comment:
@ionphractal I think `links` is deprecated: https://docs.docker.com/compose/compose-file/#links

Reply:
api is running fine on 3030 but frontend is not available on 3000

Reply:
@roschaefer Good to know, I'll take a look in a quiet minute.

@mimicc83 Grmpf that would mean it doesn't load the normal docker-compose.yml because the port is defined there and not like with api in the override file... I need tp think about how to solve this.

Reply:
@ionphractal alright thanks - if we want to run another pairing to tackle this let me know :) 
Cheers

Reply:
@ionphractal it's fine to merge `docker-compose.yml` and `docker-compose.override.yml`. We will not have different setups for local development and local staging in the near future.